assets/js/admin/job-tags-upsell.js

```diff
@@ -0,0 +1,25 @@
+const { registerPlugin } = wp.plugins
```

Comment:
Any reason to use the global instead of importing from `@wordpress/plugins`?

```suggestion
import { registerPlugin } from '@wordpress/plugins';
```


PS: We'll probably need to install `@wordpress/plugins` in the project.

Reply:
Done in 606ef3afdda5b61c3027c6f52c1611a1f66959c5